src/components/Button.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+//create button component
+function Button({name, disabled}) {
+    return (
+            <button disabled={disabled} onClick={() => {console.log(name)}}>
```

Comment:
Het valt me op dat de functie die wordt aangeroepen bij onClick in het Button component staat. Ik had dat ook gedaan. In de uitwerkingen staat de functie in de App.js. Wellicht heeft dit een reden?